.travis.yml

```
@@ -1,29 +1,56 @@
-before_install:
-  - echo -e "machine github.com\n  login $CI_USER_TOKEN" >> ~/.netrc
-language: node_js
-sudo: true
+# NOTE: Not using fs-common-build-scripts common imports, because this code is a stupid public fork
+version: ~> 1.0
+
+# Branch whitelist
+branches:
+  only:
+    - master
+# Caching of bower_components & node_modules disabled, due to an increase in delayed breakages
+cache:
+  npm: false
+  directories: null
 dist: trusty
+language: node_js
+node_js:
+  - 10
+os: linux
+
+before_install:
+  # Create .netrc for access to install private repositories
+  # NOTE: Use > (overwrite), not >> (append), because Travis CI helpfully creates one for us.
+  - 'echo -e "machine github.com\n  login $GITHUB_AUTH_TOKEN" > ~/.netrc'
+
+install:
+  - npm install -g bower
+  - bower install --config.interactive=false
+  - npm install
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+  slack:
+    # Custom template used because SUCCESS/FAILURE is all the way at the end by default
+    template:
+      - '%{result}: %{repository} <%{build_url}|#%{build_number}>  (<%{compare_url}|%{commit}>)'
+      - "\tin %{duration} for %{author}'s commit:"
+      - "\t\"%{commit_subject}\""
+    on_pull_requests: false
+    on_success: change
+    on_failure: change
+    rooms:
+      - 'familysearch:S4NvL7BI3BFGOMfoiSCfARk5#treeweb-gold-builds'
```

Since this is a public fork, I would feel more comfortable if we put this in a envar. Not that spam is likely if we don't, but It would make me feel better.
```suggestion
      - "$SLACK_ADDRESS#treeweb-gold-builds"
```

---

AFAIK, the `notifications` section does not have access to environment variables. Have you done this, before?

---

No I haven't. And if it does not then I guess I can live with it as is.